README.md

```diff
@@ -22,13 +22,13 @@
 To make this work across multiple frontends, we've separated out our logic & UI packages.
 
 - `@xstate-wizards/spells` has state machine ("spell") setup functions and references
-- `@xstate-wizards/wizards-of-react` has UI components for running in a React setup
-- `@xstate-wizards/wizards-of-vue` Placeholder for Vue implementation of HOC
```

Comment:
I made the leap of assuming HOC = Higher-Order Components, but couldn't find other references in the repo, so I simplified the language here.